Bottom: c46f27af7da7f04821d8d604cb1f6fdebe1af493
Top:    4de5887b6571c90da0520d05ea43f403a3fa34ec
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2013-08-17 01:25:07 -0300

Thread safety documentation.

for ChangeLog

	* manual/libdl.texi: New.


---

diff --git a/manual/libdl.texi b/manual/libdl.texi
new file mode 100644
index 0000000..e3fe045
--- /dev/null
+++ b/manual/libdl.texi
@@ -0,0 +1,10 @@
+@c FIXME these are undocumented:
+@c dladdr
+@c dladdr1
+@c dlclose
+@c dlerror
+@c dlinfo
+@c dlmopen
+@c dlopen
+@c dlsym
+@c dlvsym
